releasenotes/releasenotes.txt

```diff
@@ -1,31 +1,66 @@
-Exiv2 v0.27.2.1 = Exiv2 v0.27.2 RC1
+Exiv2 v0.27.2.2 = Exiv2 v0.27.2 RC2
 -----------------------------------
 
-First Release Candidate for quarterly "dot" release of Exiv2 v0.27.2
+Second Release Candidate for quarterly "dot" release of Exiv2 v0.27.2
 
 A huge thank you to Kevin for outstanding contributions to this release.
 Thank You to Dan and Luis for their solid efforts on Exiv2.
 Thank You to Andreas S for representing Team Exiv2 at LGM 2019.
-Contributors: Jens, Nehal, Phil, Simon, Toni
+Contributors: Jens, Nehal, Phil, Toni
 
 Headline Features of Exiv2 v0.27.2
 ----------------------------------
 
 1) Bug and security fixes
 2) Support for Unix (FreeBSD and NetBSD)
-3) Support for cross compiling on Linux for MinGW
-4) Support for Nikon/AutoFocus and Sony/FocusPosition Metadata
-5) Documentation and man page revisions
-6) Updated Catalan Localisation
-7) Monitoring API changes for v0.27 dot releases
-8) Using mergify to sync select PRs between 0.27-maintenance and 0.28
-
-Issues Open   for v0.27.2: https://github.com/Exiv2/exiv2/milestone/4
-Issues Closed for v0.27.2: https://github.com/Exiv2/exiv2/milestone/4?closed=1
+3) Support for Nikon/AutoFocus and Sony/FocusPosition Metadata
+4) Documentation and man page revisions
+5) Updated Catalan Localisation
+6) Monitoring API changes for v0.27 dot releases
+7) Using mergify to sync select PRs between 0.27-maintenance and 0.28
+
+Issues fixed since v0.27.1
+--------------------------
+
+#928 Corrections to README.md 2.8.Localisation
```

Comment:
This is somehow a duplication with what we have in `doc/ChangeLog`. I would just keep the list of solved issues in one place. IMHO, here it is enough to leave the previous lines:

```
Issues Open   for v0.27.2: https://github.com/Exiv2/exiv2/milestone/4	
Issues Closed for v0.27.2: https://github.com/Exiv2/exiv2/milestone/4?closed=1	
```

Reply:
I agree.  I've changed it.

About ChangeLog:  I had a script in Redmine to produce the report in ChangeLog and haven't investigated how to generate that list from GitHub.  So, it's a cut'n'paste/edit job from the web-site.  I think it's useful to have a summary.  GitHub believes there are 55 closed issues.  I think there are 35 different issues.  About 20 "to_master" duplicates.

Maybe I just stop spending time on that.  Nobody reads it!

So happy about v0.27.2  Kevin did great security work, you and Dan have done good things and our release process is becoming smoother.   Releasing v0.26 was hell.  v0.27 was tough.  v0.27.1 was OK'ish.  v0.27.2 is smooth.  v0.27.3 will be easy.